python_modules/libraries/dagster-dg/dagster_dg/cli/init.py

```diff
@@ -40,48 +51,48 @@ def init_command(
     """  # noqa: D301
     cli_config = normalize_cli_config(global_options, click.get_current_context())
 
-    workspace_name = click.prompt(
-        "Enter the name of your Dagster workspace",
-        type=str,
-        default=DEFAULT_WORKSPACE_NAME,
-    ).strip()
+    workspace_path = None
 
-    workspace_config = DgRawWorkspaceConfig(
-        scaffold_project_options=DgWorkspaceScaffoldProjectOptions.get_raw_from_cli(
-            use_editable_dagster,
+    if workspace_name:
+        workspace_config = DgRawWorkspaceConfig(
+            scaffold_project_options=DgWorkspaceScaffoldProjectOptions.get_raw_from_cli(
+                use_editable_dagster,
+            )
         )
-    )
-
-    workspace_path = scaffold_workspace(workspace_name, workspace_config)
-    workspace_dg_context = DgContext.from_file_discovery_and_command_line_config(
-        workspace_path, cli_config
-    )
 
-    project_name = click.prompt(
-        "Enter the name of your first Dagster project (or press Enter to continue without creating a project)",
-        type=str,
-        default="",
-        show_default=False,
-    ).strip()
+        workspace_path = scaffold_workspace(workspace_name, workspace_config)
 
-    if not project_name:
-        click.echo(
-            "Continuing without adding a project. You can create one later by running `dg scaffold project`."
-        )
+    if project_name is None:
+        project_name = click.prompt(
+            "Enter the name of your first Dagster project",
```

Comment:
"your _first_ Dagster project" seems inappropriate outside the workspace context. Maybe just "your Dagster project"?

Reply:
not strongly held but i thought the idea was that you ran this once to get started, so it would be the first in that context